DavidGonzalezZumaya/Collections/src/HashSet/HashSetResult.java

```diff
@@ -0,0 +1,32 @@
+package HashSet;
+
+import java.util.Scanner;
+import java.util.Set;
+import java.util.HashSet;
+
+public class HashSetResult {
+
+    public static void main(String[] args) {
+
+        Scanner in = new Scanner(System.in);
+
+        int numOfItems = in.nextInt();
+
+        String [] pairLeft = new String[numOfItems];
+        String [] pairRight = new String[numOfItems];
+
+        for (int index = 0; index < numOfItems; index++) {
+            pairLeft[index] = in.next();
+            pairRight[index] = in.next();
+        }
+
+        Set<String> pairs = new HashSet<>();
```

Comment:
It is recommended that the name of the collections and maps start with singular name and them the name of the interface, example: numberList